printerObjects.ps1

```diff
@@ -0,0 +1,19 @@
+class Snapshot {
+    #snapshot properties
+    [string]$Server;
+    [string]$Printer;
+    [int]$Action
+    [string]$Flag
+    #use class for variable validation and error control 
+    Snapshot(
+    [string]$s,
+    [string]$p,
+    [int]$a,
+    [string]$f
```

Comment:
Setting Action Constructor properties

Reply:
1st I need to set the validations for the properties and corresponding expected values.